spec/factories/collections.rb

```diff
@@ -11,5 +11,9 @@
     after(:build) do |collection, evaluator|
       collection.apply_depositor_metadata(evaluator.user.user_key)
     end
+
+    trait :public do
```

Comment:
@val99erie this seem like a strange change for this PR.  Is it meant to be a part?